tests/parser/fail_span.ml

```diff
@@ -1 +0,0 @@
-let main : "foo" = 1
```

Comment:
Can we add this test back in, just with something else? `let main : match` will be good.

Reply:
sure